### PR TITLE
Reapply IWYU fix for std::string usage in instrumentation.h

### PR DESCRIPTION
**Pull request by jdapena**
The fix applied by Stephan Hartmann adding the missing include for std::string usage in libruy has been removed again after syncing again with the copybara service.

Bug: chromium:41455655

### CODE REVIEW

**Review by talumbau — APPROVED** [review]

**Comment by jdapena** [comment]
Not sure if this is waiting for me to merge. But I don't have such permissions for this repository.

**Comment by talumbau** [comment]
I believe the bot system is working correctly. Getting the appropriate internal approvals and I hope to see if the bot can complete the action today.

**Comment by jdapena** [comment]
@talumbau any update on this? This has not been applied yet.

**Comment by jdapena** [comment]
I am not sure what is happening, but apparently the copybara change is still not landed after the parallel pull request including this change.

**Comment by jdapena** [comment]
This is still not landed. I see copybara... has tested it?, but it is not landed to master (nor is the DEPS updated in chromium).

**Pull request closed without merging**
